<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSendRecv</class>
 <widget class="QWidget" name="FileSendRecv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileSendRecv</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QLabel" name="IconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0.5px solid grey;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FilenameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>文件名(大小MB)</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>261</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right:0px;
border-radius:3px;
background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>63</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 136, 136);</string>
   </property>
   <property name="text">
    <string>传输连接中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>63</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color: #12B7F5;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecvButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>63</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color: #12B7F5;</string>
   </property>
   <property name="text">
    <string>接收</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
